core/controllers/cron.py

```diff
@@ -253,3 +253,45 @@ def get(self):
                     reviewer_ids))
             email_manager.send_mail_to_notify_contributor_dashboard_reviewers(
                 reviewer_ids, reviewers_suggestion_email_infos)
+
+
+class CronMailAdminContributorDashboardBottlenecksHandler(
+        base.BaseHandler):
+    """Handler for mailing admins if there are bottlenecks that are causing a
+    longer reviewer turnaround time on the Contributor Dashboard.
+    """
+
+    @acl_decorators.can_perform_cron_tasks
+    def get(self):
+        """Sends each admin up to two emails: an email to alert the admins that
+        there are suggestion types that need more reviewers and/or an email
+        to alert the admins that specific suggestions have been waiting too long
+        to get reviewed.
+        """
+        if not feconf.CAN_SEND_EMAILS:
+            return
+        if config_domain.NOTIFY_ADMINS_REVIEWERS_NEEDED_IS_ENABLED.value:
```

Comment:
config_domain.ENABLE_ADMIN_NOTIFICATIONS_FOR_REVIEWER_BOTTLENECKS

Reply:
@seanlip I don't know if I should make this change because it sounds like I'm enabling the whole cron handler, I like my current names because they're more specific...I can say bottlenecks for the whole thing (the handler) but can say shortage here maybe? Bottlenecks is more general. I also don't know about using stale because there's already a different get_stale_suggestions method that I didn't create in the codebase

Reply:
Fair points. I mostly don't understand your current names, so let's try and find something understandable that conveys what you want to convey (I'm mostly just guessing).

If you had no length limit on the name of this, what would you call it (precisely)? If you tell me that, I can try to help you find a shorter name.

Reply:
1. ENABLE_NOTIFYING_ADMINS_REVIEWERS_ARE_NEEDED
2. ENABLE_NOTIFYING_ADMINS_SUGGESTIONS_HAVE_WAITED_TOO_LONG_FOR_REVIEW

For 1. right now I have ENABLE_NOTIFYING_ADMINS_REVIEWER_SHORTAGE

Reply:
ENABLE_ADMIN_NOTIFICATIONS_FOR_REVIEWER_SHORTAGE seems good.

ENABLE_ADMIN_NOTIFICATIONS_FOR_SUGGESTIONS_NEEDING_REVIEW is clear, but, as you say, long. I think ENABLE_ADMIN_NOTIFICATIONS_FOR_STALE_SUGGESTIONS is probably OK; I don't think we consider "stale" to be a term of art. (I can't find get_stale_suggestions btw -- can you link to it?)

Reply:
https://github.com/oppia/oppia/blob/develop/core/storage/suggestion/gae_models.py#L307 and https://github.com/oppia/oppia/blob/develop/core/domain/suggestion_services.py#L242 I'm going to go with ENABLE_ADMIN_NOTIFICATIONS_FOR_SUGGESTIONS_NEEDING_REVIEW since it's shorter than what I have now. My other thing against using stale is that I haven't used it anywhere else in the documentation and when I first read it in the codebase I wasn't sure

Reply:
OK. I took a look at the references and I'm still fine with stale, but it's your call.